rpgJavaInterpreter-core/src/test/resources/VPARMSCALLEE.rpgle

```diff
@@ -0,0 +1,48 @@
+     D                SDS
+     D  £PDSPR           *PARMS
```

Comment:
Simplyfy this test
Since we are fixing the `*PARMS` computation issue when used in a `CALLEE` program:

- `CALLER` could pass only a param 
- `CALLEE` could have an entry with two params.

The assert in CALLEE should be  £PDSPR  = 1

For example:

**CALLER.rpgle**
```
     D P1              S             10
     
     C                   EVAL      P1='hello'
     
     C                   CALL      'CALLEE'
     C                   PARM                    P1
```

**CALLEE.rpgle**
```
     D P1              S             10
     D P2              S             10
     
     D                SDS
     D  £PDSPR           *PARMS


     C     *ENTRY        PLIST
     C                   PARM                    P1
     C                   PARM                    P2

     C     £PDSPR        DSPLY
```